eng/automation/generate_utils.py

```diff
@@ -416,6 +416,13 @@ def generate_typespec_project(
                     # clear existing generated source code, and regenerate
                     drop_changes(sdk_root)
                     remove_generated_source_code(sdk_folder, f"{group_id}.{service}")
+                    _, current_version = set_or_increase_version(sdk_root, group_id, module, version=version, preview = generate_beta_sdk)
+                    tsp_cmd.append("--emitter-options")
+                    emitter_options = f'package-version={current_version}'
+                    # currently for self-serve, may also need it in regular generation
+                    if api_version:
+                        emitter_options += f';api-version={api_version}'
```

Comment:
format for tsp-client emitter-options: `option1=value1;option2=value2`
https://github.com/Azure/azure-sdk-tools/blob/225bac743acd775fd1c6dde9cbab9c6772741e4f/tools/tsp-client/src/typespec.ts#L103